__tests__/e2e/scripts/test_integration.sh

```diff
@@ -0,0 +1,11 @@
+SYNTHETICS_E2E_ARGS="${@:2}"
+
+#!/usr/bin/env bash
+set -e pipefail
```

Comment:
Typo? Missing `-o` in order to use `pipefail` in addition no pipes are in this script, so do you think it can be simplifier with the below suggestion?

```suggestion
set -e
```